integration/kubernetes/helm-chart/alluxio/templates/fuse/daemonset.yaml

```diff
@@ -128,16 +128,16 @@ spec:
               name: {{ $val | lower }}-{{ $key | replace "_" "" }}
                           {{- end}}
                         {{- end}}
-                      {{- end}}
                       {{- else}}
-                      {{- if eq .type "hostPath"}}
-            - name: {{ .alias | lower }}
+                        {{- if eq .type "hostPath"}}
+            - name: {{ .mediumtype | lower }}
```

Comment:
is `,` allowed as part of `name`? when `mediumtype` is `MEM,SSD` what will this resolve to?

Reply:
Great question. I don't think K8s accepts comma in the name. I can address this separately in the doc.

Reply:
It's not just about the doc, multiple paths/medium types in the same tier will not work then

Reply:
we can replace this with `mount<level>` to have a unique name